admin/worker/run_autoscaler.go

```diff
@@ -2,11 +2,23 @@ package worker
 
 import (
 	"context"
+	"time"
 
+	"github.com/rilldata/rill/admin/database"
 	"github.com/rilldata/rill/admin/metrics"
 	"go.uber.org/zap"
 )
 
+const legacyRecommendTime = 7 * 24 * time.Hour
```

Comment:
What do you think of reducing this to 1 day? IMO if the pipeline has been down for several days, I would prefer we don't autoscale until it's back up.

Reply:
Yeah, that totally makes sense!

Perhaps in the future we may organize the logic here in a proper place as we are having two places to handle the data logic from this specific custom api.

Reply:
Yeah, if you prefer, feel free to refactor this to put it into the custom API if you want (e.g. the custom API could return a `stale` boolean to indicate a stale recommendation).